sphinx_click/ext.py

```diff
@@ -51,8 +51,11 @@ def _write_opts(opts):
     help = opt.help or ''
     extra = []
     if opt.default is not None and opt.show_default:
-        extra.append('default: %s' %
-                     (', '.join('%s' % d for d in opt.default) if isinstance(
+        if isinstance(opt.show_default, str):
+            extra.append('default: %s' % opt.show_default,)
```

Comment:
not: don't need the trailing comma here

Reply:
I think I put the comma there because there's a trailing comma in the other statement as well. I suspected the purpose of that to be to force it to be a tuple of one element.

But you're right it's definitely not necessary in this case because it's always a string, I'll remove the comma.